README.md

```diff
@@ -1,2 +1,51 @@
 # in-stock-bot
-A simple python telegram bot checking if a product is in stock
+* [![GitHub Stars](https://img.shields.io/github/stars/MayNiklas/in-stock-bot.svg)](stargazers)
+
+* [![GitHub Issues](https://img.shields.io/github/issues/MayNiklas/in-stock-bot.svg)](issues)
+
+* [![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)
+
+* [![Telegram Chat](https://img.shields.io/badge/Telegram-Join%20to%20us-blue.svg)](TelegramChannel)
```

Comment:
This one did not work for me.
I don't know if I personally would like to link the telegram channel.
If you want to: we can do so!
What do you think?
I personally would leave it out.

Reply:
I will gladly remove the link.